languages/de.yml

```diff
@@ -0,0 +1,19 @@
+categories: Kategorien
+search: Suche
+tags: Tags
+tagcloud: Tag Cloud
+tweets: Tweets
+prev: zurück
+next: weiter
+comment: Kommentare
+archive_a: Archiv
```

Comment:
`Archive` instead of `Archiv`?
Sorry, I'm not familiar with German, maybe I confused something.